### DISCUSSION TITLE
50

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#768] [Comments]
http://jojoldu.tistory.com/50

**Comment by jojoldu** [comment]
작성자: seungdols

제가 이틀전에 샀는데!!ㅎㅎ 역시 좋은 개발자분들이 보시는 책이라는 것이 틀림 없군요!!^^

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

ㅎㅎ seungdols님도 좋은 개발자라는 것 아닐까요!? ㅎㅎ 항상 블로그 방문해주셔서 감사합니다^^

**Comment by jojoldu** [comment]
작성자: 전정완

좋은 글 감사합니다! 다른 에디터에서 마크다운으로 작성한 글을 어떻게 티스토리로 바로 업로드 하셨는지 여쭈어봐도 될까요?

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

방문해주셔서 감사합니다^^
저같은 경우엔 마크다운으로 작성한 파일을 github에 업로드하고, github에서 출력되는 화면의 html 소스를 복사해서 그대로 티스토리에 붙여넣기하여 사용합니다.
이방식은 이미지만 티스토리에서 재등록하면되서 사용하고있습니다.
더 좋은 방법이 있는지 아직까진 발견못한 상황입니다^^;

**Comment by dharana77** [comment]
이책 좋아보이는데 너무 오래되서 지금사도 괜찮을까요?_? 크게 바뀐 내용은 없는지 real my sql이 나을지 궁금합니다!
